app/src/main/java/com/capstone/weatherapp/City.kt

```diff
@@ -0,0 +1,15 @@
+package com.capstone.weatherapp
+
+data class City(val coord: Coord, val weather: List<Weather>, val base: String, val main: Main,
```

Comment:
let's clean this data class up. each smaller data class doesn't need to be nested inside City, they can all be at the same level